spec.bs

```diff
@@ -221,6 +226,55 @@ The <dfn for="interest group">estimated size</dfn> of an [=interest group=] |ig|
     [=interest group ad/render url=].
   1. The [=string/length=] of |ad|'s [=interest group ad/metadata=] if the field is not null.
 
+<div algorithm>
+
+To <dfn>check interest group permissions</dfn> given an [=url/origin=]
+|ownerOrigin|, an [=url/origin=] |frameOrigin|, and a [=boolean=] |isJoin|:
+1. If |ownerOrigin| is [=same origin=] to |frameOrigin|, then return true.
+1. Let |permissionsUrl| be the result of [=building an interest group permissions url=]

```

Comment:
not sure whether we would like to spec the caching of .well-known fetching results. So probably add something like "1. TODO: maybe cache .well-known fetching results and check cache first".

Reply:
My hunch is that caching is an implementation detail, and need not be in the spec.

That said, the explainer has a note about always fetching the well-known URL regardless of whether the user-agent is in the interest group, so I will add that here.

Reply:
yeah that's fine. I'm not sure whether it should be in spec or not, neither. Sometimes it's hard for me to determine whether it's an implementation detail.

Reply:
As discussed over chat, if our implementation is just deferring to the usual HTTP cache semantics, then we don't have to mention anything here. It mostly matters if we're doing some special out-of-band caching that might have other web-observable effects peculiar to this instance